Taskfile.yml

```diff
@@ -729,7 +729,7 @@ tasks:
         docker compose run --rm textlint --fix --config build/.textlintrc --ignore-path .prettierignore
         "**/*.md" "**/*.mdx" ".github/**/*.md"
       - >
-        docker compose run --rm prettier --write --parser markdown --no-semi --single-quote --trailing-comma none
+        docker compose run --rm prettier --write --parser markdown --single-quote --semi --trailing-comma none
```

Comment:
I still don't understand why we want to add `--semi` option, `--no-semi` seemed fine :thinking: